_uw-research-computing/htc-docker-to-apptainer.md

```diff
@@ -0,0 +1,93 @@
+---
+highlighter: none
+layout: guide
+title: "Convert Docker images to Apptainer images"
+guide: 
+    order: 8
+    category: Software Solutions
+    tag:
+        - htc
+---
+
+{% capture content %}
+
+- [Why convert your Docker image to an Apptainer image?](#why-convert-your-docker-image-to-an-apptainer-image)
+- [Convert a Docker image into an Apptainer image](#convert-a-docker-image-into-an-apptainer-image)
+   * [Move the image to your staging directory](#move-the-image-to-your-staging-directory)
+- [Use the Apptainer image in a job](#use-the-apptainer-image-in-a-job)
+
+{% endcapture %}
+{% include /components/directory.html title="Table of Contents" %}
+
+
+## Why convert your Docker image to an Apptainer image?
+
+While HTCondor can pull container images from Docker Hub, we recommmend converting Docker images to local Apptainer images (`.sif` files) so that you have a local copy of your container image. This reduces pull requests from Docker Hub.
+
+## Convert a Docker image into an Apptainer image
+
+We will need to launch an interactive job to build the Apptainer image. Below is an example of a submit file. You may want to change memory or disk space requirements, depending on the expected requirements for your image.
+
+```
+# build.sub
+# For building an Apptainer container
+
+log = build.log
+
+# In the latest version of HTCondor on CHTC, interactive jobs require an executable.
+# If you do not have an existing executable, use a generic linux command like hostname as shown below.
+executable = /usr/bin/hostname
+
```

Comment:
```suggestion
```

Reply:
Not only has the bug been patched, we no longer need to specify the executable file. 
(You may want to confirm this before merging)